### DISCUSSION TITLE
Problem With Page Transitions

### DISCUSSION THREAD

**Discussion by Neography7** [inertiajs/inertia#1065] [Help]
I have a problem with page transitions. 

My routes are like this;

site.dev/
site.dev/news/{slug}
site.dev/{username}

When clicked user link at news page inertia js routing to "news/{username}" but it normally it should route "{username}. The strange thing is when I clicked the link with the middle button on the mouse it opens with normally.

`<Link :href="user.username">{{user.username}}</Link>`

I set the base href tag on head tag but it didn't solve.

How I can fix this error?

**Comment by ajnsn** [comment]
Hey @Neography7 
You probably should put the root-slash before your :href="user.username" as relative url's will be -relative- to their parent, e.g. `/news/{username}` when users are on the `/news-page`. 

```
<Link :href="`/${user.username}`">{{user.username}}</Link>
```

Does this work?

**Reply by Neography7** [reply]
Yes it works but is it like how supposed to be?
